.github/workflows/build-images-workflow-run.yml

```diff
@@ -182,7 +182,7 @@ jobs:
     needs: [cancel-workflow-runs]
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
Can you tell more about it?

Reply:
Yep. It was added to the commit which I updated before but I have not updated the PR description. Synced now

Reply:
Also, see the comment below :)

Reply:
In short workfow_run always runs from master - no matter which is  the branch the PR originates from, so we have to have all the python version supported there, but I exclude those, that are not needed for the given "branch" (I base it on the DEFAULT_BRANCH which is in the incoming PR) -> if DEFAULT_BRANCH = master -> 3.6, 3.7, 3.8 are build, otherwise all five.

Reply:
That's the dark side of "workffow_run" flow. We have to make sure that scripts in "master" can be used to build all the images that are used in all other branches, that's a bit annoying, but that's the price to pay for optimisation of the builds and "security".